distributed/asyncio.py

```diff
@@ -127,10 +127,6 @@ def __init__(self, *args, **kwargs):
     async def __aexit__(self, type, value, traceback):
         await self.shutdown()
 
-    def __del__(self):
-        if self.status == 'running':
-            self.loop.asyncio_loop.run_until_complete(self.shutdown(fast=True))
```

Comment:
Here's an experiment that does what you propose:

https://github.com/ajdavis/distributed/pull/1

It passes Travis and has no AssertionErrors logged from destructors. Is it worth doing? Does adding a `__del__` to a class in Python 3 put any more pressure on the garbage collector or is that problem completely solved?